test/models/linkset_test.rb

```diff
@@ -22,6 +22,12 @@ class LinksetTest < ActiveSupport::TestCase
       end
       assert_empty @linkset
     end
+
+    should "enqueue a job to verify linkbacks" do
+      assert_enqueued_with(job: VerifyLinkbacksJob, args: [@linkset.rubygem_id]) do
```

Comment:
It looks like this is failing in CI. I think you're missing `include ActiveJob::TestHelper`  in the class.

```
require "test_helper"

class LinksetTest < ActiveSupport::TestCase
  include ActiveJob::TestHelper
# ...
```

Reply:
I moved the `assert_enqueued_with` test to `VerifyLinkbacksJobTest` since it's more of a job thing. :D